pytorch_to_returnn/torch/nn/modules/shape.py

```diff
@@ -368,8 +368,9 @@ def _get_output_shape_from_returnn(self,
 class UnflattenBatch(Module):
   is_original_torch_module = False
 
-  def __init__(self):
+  def __init__(self, batch_first: Optional[bool] = None):
```

Comment:
Also here, maybe it should not be optional. Maybe this should always be explicit. Because on Torch side, it will always matter which way it is.

Reply:
The advantage of allowing `None` would be that we would get the order in RETURNN that was used before flattening. But yes, on Torch side it's always explicit so I'm fine with being explicit here as well.